source/_includes/installation/core.md

```diff
@@ -102,3 +102,22 @@ If this address doesn't work you may also try `http://localhost:8123` or `http:/
 When you run the `hass` command for the first time, it will download, install and cache the necessary libraries/dependencies. This procedure may take anywhere between 5 to 10 minutes. During that time, you may get a **site cannot be reached** error when accessing the web interface. This will only happen the first time. Subsequent restarts will be much faster.
 
 </div>
+
+### Enable shell auto-completion
+
+Bash or ZSH auto-completion features can be used with the `hass` command.
+For example, if you use bash, you can add the following to your `~/.bashrc` file:
+
+```bash
+eval "$(register-python-argcomplete hass)"
+```
```

Comment:
Since you mention ZSH here it would be good to include an example for it.

Reply:
I've committed instructions for ZSH, also for tcsh and fish.
Source: [pipx documentation](https://pypa.github.io/pipx/installation/#shell-completion) then instruction when running `pipx completions`.

Reply:
Looks great, thanks! :)